frontend/src/lib/components/HelpButton/HelpButton.tsx

```diff
@@ -139,14 +140,27 @@ export function HelpButton({
                             </LemonButton>
                         </a>
                     )}
+                    {!isPromptVisible && (
+                        <LemonButton
+                            icon={<IconTrendUp />}
```

Comment:
Hmm, this icon is super old, I think `IconTrendingUp` is our more modern version